### ISSUE TITLE
Locations - Bibliographical references in the Graffiti field

### ISSUE THREAD

**Issue by mparani70** [MedCyprus/MedCyprusEFES#30]
Display of bibliographical references in the Graffiti field. Currently, only the cited range appears, but not a link to the relevant bibliographic entry.

**Comment by emylonas** [comment]
(Notes for EM)
Some of the monuments cite graffiti with a bibliographic reference
```
<desc type="graffiti">Yes (<bibl><ptr target="bibliography.xml#Demestichaetal2017"/><citedRange>376</citedRange></bibl>)</desc>
```
(See number 2)
and some insert a URL to "historic graffiti" directly 
```
<desc type="graffiti">https://historic-graffiti.cyi.ac.cy/place/6/</desc>
```
(See number 21)
currently, the indexing XSLT handles the second case, but is unaware of the first case. Will add a test for which format, and the code for bibliography. 
Curious if the word "Yes" is necessary. It only appears together with a bibliographic citation, not the Historic Graffiti URL.

**Comment by emylonas** [comment]
Email thread on this issue added so it's all in one place! 
From Maria (2024-10-12)
Good-morning.  Since we are using "Presence not verified", perhaps "Presence confirmed" instead of simply "Present" might be an alternative. At Gialia (#16 and #17), because this is somewhat exceptional, I thought to indicate that the graffiti are in Georgian. But, now that you pointed it out and I thought about it, we have graffiti in other languages (e.g. Latin) in other monuments, and this is not indicated, since we are not going into details. So, for the sake of consistency, Georgian should be removed.

From Elli (2024-10-12)
Maria, thank you! It's fine to keep the "Yes", although something like "Present" might be clearer. In one or two cases, there is also some further information, #16 and #17 indicate that they are in Georgian. 

I am also asking because I'm trying to make the bibliographic references show up, and need to take into account all the other components that might appear.
From Maria (2024-10-14)
Good-morning.  Since we are using "Presence not verified", perhaps "Presence confirmed" instead of simply "Present" might be an alternative. At Gialia (#16 and #17), because this is somewhat exceptional, I thought to indicate that the graffiti are in Georgian. But, now that you pointed it out and I thought about it, we have graffiti in other languages (e.g. Latin) in other monuments, and this is not indicated, since we are not going into details. So, for the sake of consistency, Georgian should be removed.

**Comment by emylonas** [comment]
Bibliographic references are displaying correctly. 
In order for this to process properly, it's necessary to make some a change in the markup of graffiti. I will add this to the MedCyprus documentation. 
- URLs, such as the Cypriot graffiti reference should be encoded as pointers eg `<ptr target="https://historic-graffiti.cyi.ac.cy/place/1/"/>` so that they can easily be differentiated from plain text. 
- ideally, there will only be plain text at the beginning of a graffito entry. 
-don't put any punctuation around bibliography such as parentheses around the citations or semi colons in between. These will be supplied by the XSLT. 

Long description, for the record: 
I made some new Solr fields in order to handle the parts of a graffiti entry more easily. One each for the plain text (index_graffiti_text), one for a URL (index_graffiti_ptr) and one for bibliographic references encoded using the `<bibl>` element (index_graffiti_bibl). These are handled separately in indices.xsl

Not closed because not pushed yet.

**Comment by emylonas** [comment]
Fixed with f565f7ac1d6a8fbd0d1c0ae7a728389beadfdfad
